esda/moran.py

```diff
@@ -1760,18 +1758,19 @@ def _explore_local_moran(moran_local, gdf, crit_value, explore_kwargs=None, m=No
         geodataframe used to create the Moran_Local class
     crit_value : float, optional
         critical value for determining statistical significance, by default 0.05
-    explore_kwargs : dict, optional
-        additional keyword arguments passed to geopandas.explore, by default None
-    m : folium.Map, optional
-        folium map canvas to plot on top of, by default None
+    kwargs : dict, optional
+        additional keyword arguments are passed directly to geopandas.explore, by default None
 
     Returns
     -------
     m
         folium.Map
     """
-    if explore_kwargs is None:
-        explore_kwargs = dict()
+    if kwargs is None:
+        kwargs = dict()
+    if 'cmap' in kwargs:
```

Comment:
I would a a warning here. People may be wondering why a custom cmap makes no difference.

Reply:
good point. Actually, lets just give control of the cmap back and set this as default. (when you have all four (so, five) categories, they actually end up in the proper order, so something like RdBu could make sense here if somebody wants to override